include/velocypack/Slice.h

```diff
@@ -117,38 +117,70 @@ class Slice {
   // creates a slice of type MaxKey
   static constexpr Slice maxKeySlice() noexcept { return Slice(maxKeySliceData); }
 
-  // pointer to the head byte
-  constexpr uint8_t const* start() const noexcept { return _start; }
-  
+  int64_t getTagId() const {
+    if(isTagged()) {
+      if(*_start == 0xee) {
+        return readIntegerFixed<ValueLength, 1>(_start + 1);
+      } else if(*_start == 0xef) {
+        return readIntegerFixed<ValueLength, 8>(_start + 1);
+      } else {
+        throw new Exception(Exception::InternalError, "Invalid tag type ID");
+      }
+    }
+
+    return 0;
+  }
+
+  constexpr uint8_t tagOffset() const {
+    return tagOffset(rawStart());
+  }
+
+  // pointer to the head byte, including possible tags
+  constexpr uint8_t const* rawStart() const noexcept {
+    return _start;
```

Comment:
what about adding changing this to `_start + getTagOffset()` instead, so `rawStart()` can be used to get the VPack value without tag information? This also means the `noexcept` specifier needs to be removed here.